src/Calculator.Shared/Views/MainPage.xaml

```diff
@@ -61,7 +61,7 @@
 		</Style>
 	</Page.Resources>
 
-	<Grid toolkit:VisibleBoundsPadding.PaddingMask="All">
```

Comment:
This shouldn't be removed

Reply:
Right, I'll revert this. However I had to do that because it does not run on Windows, did you try to run the Windows app when you add it ?

Reply:
I ran the app on Windows 30 s ago and it worked. Did you have any local changes to the UWP head? It needs to have the Uno.UI NuGet installed.

Reply:
Nope. Maybe that's only because I forgot to restore packages after last pull ...